e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/DemographicsTest.groovy

```diff
@@ -51,17 +51,20 @@ class DemographicsTest extends Specification {
 
         when:
         def response = demographicsClient.submit(invalidJsonRequest, true)
-        def responseBody = EntityUtils.toString(response.getEntity())
+        def parsedJsonBody = JsonParsing.parseContent(response)
 
         then:
         response.getCode() == 400
+        !StringUtils.isEmpty(parsedJsonBody.error as String)
```

Comment:
This seems like a weird helper: `org.apache.groovy.parser.antlr4.util.StringUtils`.  Especially since this seems to be some internal-ish helper associated with ANTLR.  You can do the same natively in Groovy by doing the following.
```suggestion
        !(parsedJsonBody.error as String).isEmpty()
```

Reply:
Good point, I didn't notice it was coming from antlr, I thought it was internal. I'll use your suggestion, that's what I was looking for